src/workerd/jsg/jsg.h

```diff
@@ -2063,6 +2103,73 @@ class Lock {
   ContextScope enterContextScope(v8::Local<v8::Context> context);
   // Ensures that we are currently within the scope of the given v8::Context
 
+  // ====================================================================================
+  JsValue undefined() KJ_WARN_UNUSED_RESULT;
+  JsValue null() KJ_WARN_UNUSED_RESULT;
+  JsBoolean boolean(bool val) KJ_WARN_UNUSED_RESULT;
+  JsNumber num(double) KJ_WARN_UNUSED_RESULT;
+  JsNumber num(float) KJ_WARN_UNUSED_RESULT;
+  JsInt32 num(int8_t) KJ_WARN_UNUSED_RESULT;
+  JsInt32 num(int16_t) KJ_WARN_UNUSED_RESULT;
+  JsInt32 num(int32_t) KJ_WARN_UNUSED_RESULT;
+  JsUint32 num(uint8_t) KJ_WARN_UNUSED_RESULT;
+  JsUint32 num(uint16_t) KJ_WARN_UNUSED_RESULT;
+  JsUint32 num(uint32_t) KJ_WARN_UNUSED_RESULT;
+  JsBigInt bigInt(int64_t) KJ_WARN_UNUSED_RESULT;
+  JsBigInt bigInt(uint64_t) KJ_WARN_UNUSED_RESULT;
+  JsString str() KJ_WARN_UNUSED_RESULT;
+  JsString str(kj::ArrayPtr<const char16_t>) KJ_WARN_UNUSED_RESULT;
+  JsString str(kj::ArrayPtr<const uint16_t>) KJ_WARN_UNUSED_RESULT;
+  JsString str(kj::ArrayPtr<const char>) KJ_WARN_UNUSED_RESULT;
+  JsString str(kj::ArrayPtr<const kj::byte>) KJ_WARN_UNUSED_RESULT;
+  JsString strIntern(kj::StringPtr) KJ_WARN_UNUSED_RESULT;
+  JsString strExtern(kj::ArrayPtr<const char>) KJ_WARN_UNUSED_RESULT;
+  JsString strExtern(kj::ArrayPtr<const uint16_t>) KJ_WARN_UNUSED_RESULT;
+  JsSymbol symbol(kj::StringPtr) KJ_WARN_UNUSED_RESULT;
+  JsSymbol symbolShared(kj::StringPtr) KJ_WARN_UNUSED_RESULT;
+  JsSymbol symbolInternal(kj::StringPtr) KJ_WARN_UNUSED_RESULT;
+  JsObject obj() KJ_WARN_UNUSED_RESULT;
+  JsObject map() KJ_WARN_UNUSED_RESULT;
+  JsValue external(void*) KJ_WARN_UNUSED_RESULT;
+  JsValue error(kj::StringPtr message) KJ_WARN_UNUSED_RESULT;
+  JsValue typeError(kj::StringPtr message) KJ_WARN_UNUSED_RESULT;
+  JsValue rangeError(kj::StringPtr message) KJ_WARN_UNUSED_RESULT;
+  JsDate date(double timestamp) KJ_WARN_UNUSED_RESULT;
+  JsDate date(kj::Date date) KJ_WARN_UNUSED_RESULT;
+  JsDate date(kj::StringPtr date) KJ_WARN_UNUSED_RESULT;
```

Comment:
Many of these functions, here and also those below, appear to be untested.

Testing some of these one-liners feels like overkill, but it's probably worth it.